<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandPanel</class>
 <widget class="QWidget" name="CommandPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="commandStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="portPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollAreaSerialCommands">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContentsSerial">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>537</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="blePage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollAreaBLECommands">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContentsBLE">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>537</width>
            <height>73</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbNew">
     <property name="text">
      <string>New Command</string>
     </property>
     <property name="icon">
      <iconset resource="../misc/winicons.qrc">
       <normaloff>:/icons/tango/32x32/actions/list-add.png</normaloff>:/icons/tango/32x32/actions/list-add.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../misc/winicons.qrc"/>
 </resources>
 <connections/>
</ui>
